CRYOSMSApp/src/CRYOSMSDriver.cpp

```diff
@@ -52,33 +52,30 @@ return status; \
 #define RETURN_IF_ASYNERROR1(func, arg) status = (func)(arg); \
 if (status != asynSuccess)\
 {\
-errlogSevPrintf(errlogMajor, "Error returned when calling %s with arguments %s", #func, arg);\
+errlogSevPrintf(errlogMajor, "Error returned when calling %s", #func);\
 return status; \
 }
 
 #define RETURN_IF_ASYNERROR2(func, arg1, arg2) status = (func)(arg1, arg2); \
 if (status != asynSuccess)\
 {\
-errlogSevPrintf(errlogMajor, "Error returned when calling %s with arguments %s", #func, arg1);\
+errlogSevPrintf(errlogMajor, "Error returned when calling %s", #func);\
```

Comment:
Would it work to use e.g.  `std::to_string(arg1).c_str()` ? I have a feeling `to_string` may fail when passed a string.  You'd then need to create your won template function that called to_string except for the specised case of being passed a string when it does nothing. another option is to make the macro create a temporary scoped string stream and then print this output.